clients/src/test/java/org/apache/kafka/clients/producer/internals/TransactionManagerTest.java

```diff
@@ -2980,6 +2980,68 @@ public void testEpochBumpAfterLastInflightBatchFails(boolean transactionV2Enable
         assertEquals(0, transactionManager.sequenceNumber(tp0));
     }
 
+    @ParameterizedTest
+    @ValueSource(booleans = {true})
```

Comment:
true,false?

Reply:
This was just to check in the case that V2 was enabled if the epoch bump would be updated on the client, I can maybe remove the parameterization since we only want to check for v2?

Reply:
If we removed the parameterization, can we just make it @Test